files/__addonLocation__/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "@tsconfig/ember/tsconfig.json",
+  "compilerOptions": {
+    "paths": {
+      // No paths, no absolute imports, only node_modules imports allowed
+      // But fallback for type-overrides and such
+      "*": ["types/*"]
+    }
+  },
+  "include": [
+    "src/**/*",
+    "types/**/*"
+  ],
+  "glint": {
+    "environment": "ember-loose",
```

Comment:
why loose?
porque no template-imports?

tho :thinking: maybe that should be done as a part of https://github.com/embroider-build/addon-blueprint/issues/41

Reply:
As far as I know it isn't even possible to use FCCTs in v2 addons yet, so including the `template-imports` environment here would be a pretty mean trick to play on authors 😜